.github/workflows/full_test.yaml

```diff
@@ -0,0 +1,25 @@
+name: Full Test
+
+on:
+  schedule:
+    - cron: "12 6 * * 1-5"
+
+env:
+  GO_VERSION: 1.15
+
+jobs:
+  go:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+        with:
+          fetch-depth: 1
+      - uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: running tests
+        run: |
+          find . -name "*_test.go" | \
+            xargs -n 1 dirname | sort | uniq | \
+            xargs -n 1 go test -count=1 -race
```

Comment:
```suggestion
        run: go test -race ./...
```

Reply:
doesn't work because the tests open actual ports which make the tests fail. Even `go test -race -count=1 ./...`, my first attempt, fails... This is why I use the `xargs -n 1` to only supply one directory at a time to `go test`, else the different packages are tested in parallel, which, fails...

Reply:
```suggestion
        run: go list -f '{{.Dir}}' ./... | xargs -n1 go test -race
```

Reply:
that will also test directories that have no tests in them...

Reply:
Correct, and for each "untest" directory, `go test` outputs smth like `?   	go.dedis.ch/cothority/v3/authprox/apadmin	[no test files]`, without failing.

My point is that it makes it more readable to use `go list` rather than using 4 pipes.